### PR TITLE
Handle cancelled proposals, add new config attributes.

### PR DESCRIPTION
**Pull request by fabriziovigevani**
Updates subgraph according to https://github.com/1Hive/conviction-voting-app/pull/82/files.

TODO:
- [x] Update connector once https://github.com/1Hive/conviction-voting-app/pull/89 is merged

### CODE REVIEW

File: connector/subgraph/src/ConvictionVoting.ts [hunk 1]
```diff
@@ -13,6 +14,9 @@ import {
   getStakeHistoryEntity,
   getOrgAddress,
 } from './helpers'
+import { STATUS_CANCELLED, STATUS_EXECUTED } from './proposal-statuses'
```
Comment (rperez89) [inline connector/subgraph/src/ConvictionVoting.ts:17]: shouldn't we just import `ProposalStatus` enum from the schema and use what we have there?

File: connector/subgraph/src/helpers.ts [hunk 2]
```diff
@@ -8,6 +8,7 @@ import {
 } from '../generated/schema'
 import { MiniMeToken as MiniMeTokenContract } from '../generated/templates/MiniMeToken/MiniMeToken'
 import { ConvictionVoting as ConvictionVotingContract } from '../generated/templates/ConvictionVoting/ConvictionVoting'
+import { STATUS_ACTIVE } from './proposal-statuses'
```
Comment (rperez89) [inline connector/subgraph/src/helpers.ts:11]: same here

File: connector/subgraph/src/proposal-statuses.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const STATUS_ACTIVE = 'Active'
```
Comment (rperez89) [inline connector/subgraph/src/proposal-statuses.ts:1]: and remove this file if what i am saying works

**Comment by rperez89** [comment]
Ah now that i am thiking if we take a look to the `_updateProposalStakes` we are always pushing a new stake regardless if the user withdrawn all his previous stake, and now since we need the stakers per proposal we might want to delete from `proposal.stakes` if the amount after withdraw is 0 right?

File: connector/subgraph/src/ConvictionVoting.ts (hunk 1 above)
Comment (fabriziovigevani) [reply connector/subgraph/src/ConvictionVoting.ts:17]: Don't think there's a way to import the declared enums from the schema. Do you know any?

**Comment by fabriziovigevani** [comment]
As discussed offline, the stakedAmount added to the stake entity is the resulted stake after the user staked or withdrawn.

**Pull request merged**